tests/main/preseed/task.yaml

```diff
@@ -42,6 +42,12 @@ restore: |
   umount_ubuntu_image "$IMAGE_MOUNTPOINT" || true
 
 execute: |
+  if os.query is-focal; then
```

Comment:
Maybe a small comment here ? it's a bit counter-intuitive that focal (20.04) is using core18, maybe some context (if possible) why 20.10+ is using core20

Reply:
I think it's lxd changing base, these images have lxd in them

Reply:
maybe we can extract the info from the model inside them at some point, not sure